conf/mig-agent-conf.go.inc

```diff
@@ -156,3 +156,6 @@ MIIEpAIBAAKCAQEAvJQqCjE4I63S3kR9KV0EG9e/lX/bZxa/2QVvZGi9/Suj65nD
 ........
 RMSEpg+wuIVnKUi6KThiMKyXfZaTX7BDuR/ezE/JHs1TN5Hkw43TCQ==
 -----END RSA PRIVATE KEY-----`)
+
+//PERSISTENTMODULES contains module names and params that are to be started when the agent starts initially
+var PERSISTENTMODULES = [][]string {{"audit",`{"class":"parameters", "parameters":{"rulefilepath": "audit.rules.json", "outputsockets": ["/path/to/unix.sock"], "serveraddress": "mozdef_url"}}`}}
```

Comment:
We have a socket path option here and a server address; how does it decide which is used? is there a way to specify a subset of output modes to use or select a specific one?